lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,57 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleBundleDefinitionSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ * <p>
+ * <b>Module bundle section modes</b><br>
+ * <ul>
+ * 	<li>
+ * 		<code>provided</code>: A section of type 'provided' defines a set of modules that should not be included in the bundle file itself, but
```

Comment:
So is it called embedding `technology`, 'mode' or `type`?
This could be made a bit more consistent. I would opt for `mode` as this is the actual property in the API.

Reply:
Done